src/cli/java/org/commcare/util/screen/MultiSelectEntityScreen.java

```diff
@@ -87,6 +87,15 @@ protected void updateSession(CommCareSession session) {
         }
     }
 
+    @Override
+    public boolean referencesContainStep(String stepValue) {
```

Comment:
Where will this get used?

Reply:
This gets called when [we rebuild the menu session from session frame](https://github.com/dimagi/formplayer/blob/master/src/main/java/org/commcare/formplayer/services/MenuSessionFactory.java#L90) after EOF navigation or case claim to establish the validity of the datum in the session frame.